[Monitor][Query] Support non-public clouds (#29602)
Azure/azure-sdk-for-python: sdk/monitor/azure-monitor-query/README.md

async_metrics_client = MetricsQueryClient(credential)
```

#### Configure clients for non-public Azure clouds

> By default, `LogsQueryClient` and `MetricsQueryClient` are configured to connect to the public Azure cloud. These can be configured to connect to non-public Azure clouds by passing in the correct `endpoint` argument: For example:

Reviewer: ```suggestion
By default, `LogsQueryClient` and `MetricsQueryClient` are configured to connect to the public Azure cloud. These clients can be configured to connect to non-public Azure clouds via the `endpoint` argument: For example:
```
